packages/react-components/react-breadcrumb/src/utils/truncateLongContent.ts

```diff
@@ -0,0 +1,21 @@
+const MAX_NAME_LENGTH = 30;
+const MAX_TOOLTIP_LENGTH = 80;
+
+export const truncate = (content: string, maxLength: number): string => {
+  return content.length > maxLength
+    ? content
+        .trim()
+        .slice(0, maxLength - 3)
+        .concat('...')
+    : content;
+};
+
+export const truncateLongName = (content: string, maxLength?: number): string => {
```

Comment:
These two methods are the same `truncateLongName` and `truncateLongTooltip` but they only use different default length value. I was thinking, maybe it would be good to have a length as a type. But it's just an idea. The way it is can can be ok